components/CDPUserDashboard.tsx

```diff
@@ -109,36 +153,45 @@ const CDPUserDashboard = (props: Props): JSX.Element => {
     // Negative sorts "reverse alphabetically" which allows newest files to appear first
     .sort((a: CDPFile, b: CDPFile) => -a.key.localeCompare(b.key))
 
+  const isSurveyResponse = (wanted: boolean) => (file: CDPFile) =>
+    file.key.includes('survey-response') === wanted
+  const surveyResponses = files.filter(isSurveyResponse(true))
```

Comment:
It might be nice to have a filter to only show merged responses. The sort might make that unneeded though, at least for now.